duckduckgo/macos-browser
# [WIP] dependency injection

Task/Issue URL:
Tech Design URL:
CC:

**Description**:
- Tests support TBD

**Steps to test this PR**:
1. Make sure the project can be run both under Xcode 14.3.1 and Xcode 15 beta
2. Validate standard Release Template test cases



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
